### PR TITLE
Parameter to specify additional client info

### PR DESCRIPTION
**Pull request by johnrogers**
Hi all, me again!

We're trying to use Google Cloud Functions to do stuff like output to Slack once the test job has finished. We run our tests asynchronously to save on CI minutes, so waiting for the result locally is not going to work for us.

It appears that the only way to get any sort of metadata out at the other end (e.g. into the cloud function when the test job has finished), is to specify it In the `ClientInfo` information when submitting the job: https://firebase.google.com/docs/reference/functions/functions.testLab.ClientInfo

I've added a parameter that will allow the user to specify additional client info.

DISCLAIMER: I am not great at Ruby, there is probably a better way to do this.

### CODE REVIEW

**Comment by johnrogers** [comment]
@powerivq @MartyCatawiki thoughts?

**Comment by MartyCatawiki** [comment]
@johnrogers I'm just using the `fastlane` `slack` feature when it's all ready. That works good enough for me. @powerivq what are your thoughts? And can you also please take a look at my PR?

File: lib/fastlane/plugin/firebase_test_lab/helper/ftl_service.rb [hunk 1]
```diff
@@ -80,7 +80,15 @@ def get_default_bucket(gcp_project)
         end
       end
 
-      def start_job(gcp_project, app_path, result_path, devices, timeout_sec)
+      def start_job(gcp_project, app_path, result_path, devices, timeout_sec, additional_client_info)
+
```
Comment (powerivq) [inline lib/fastlane/plugin/firebase_test_lab/helper/ftl_service.rb]: Nit: no empty line here

**Comment by johnrogers** [comment]
@powerivq Fixed and force-pushed.

@MartyCatawiki - Our OS X CI box minutes are expensive compared to a Linux box. We're running the tests asynchronously so we don't get the reply to be able to `slack` from `fastlane`.

**Comment by johnrogers** [comment]
@powerivq Thanks for the merge. Can you tag this as 1.0.6 please?

**Comment by johnrogers** [comment]
@powerivq How about that tag?

**Pull request merged**